### ISSUE TITLE
partial cancellation document has no A11y version

### ISSUE THREAD

**Issue by MalteJanz** [shopware/platform#10088]
### Shopware Version

6.7.0 RC5 cloud

### Affected area / extension

Platform(Default)

### Actual behaviour

No link in mail to HTML version of document for "partial cancellation invoice" like it is the case for other documents like "invoice" or "delivery note"

![Image](https://github.com/user-attachments/assets/9ea85610-85b7-43db-be09-80498beed567)

### Expected behaviour

It should have a HTML / A11y version as well

### How to reproduce

- Make an order
- Generate invoice
- Add return items
- Generate cancellation invoice
- Look in mail for A11y version (there is none)

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.
